### ISSUE TITLE
`no-missing-imports` does not support "exports" in package.json

### ISSUE THREAD

**Issue by giltayar** [mysticatea/eslint-plugin-node#255]
When checking for existence of an import, the new "exports" field is not checked.

**Comment by david-bojnansky** [comment]
There can be also `imports` field in `package.json` which can be used for mappings.

```json
"imports": {
    "#abc/*": "./abc/out/*"
}
```

```js
import x from '#abc/file'
```

**Comment by BePo65** [comment]
Any progress on this? The missing support of "exports" e.g. creates a false positive when using [got](https://github.com/sindresorhus/got) for fetching web resources.
